platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/SyncManagerTest.java

```diff
@@ -88,13 +89,13 @@ public SyncManagerTestData(final SwirldStateManager swirldStateManager) {
             startUpEventFrozenManager = mock(StartUpEventFrozenManager.class);
             final Random random = getRandomPrintSeed();
             hashgraph = new DummyHashgraph(random, 0);
-            final TransactionConfig transactionConfig =
-                    new TestConfigBuilder().getOrCreateConfig().getConfigData(TransactionConfig.class);
-            eventTransactionPool = spy(new EventTransactionPool(new NoOpMetrics(), transactionConfig, null));
+            final PlatformContext platformContext =
+                    TestPlatformContextBuilder.create().build();
+
+            eventTransactionPool = spy(new TransactionPool(platformContext, null));
```

Comment:
nit: member name could be changed to `transactionPool`

Reply:
updated